src/commands/install-mise.yml

```diff
@@ -0,0 +1,21 @@
+description: >
+  Installs mise tool version manager
```

Comment:
Hmm I'm wondering if we should add these commands in `purchases-ios` instead of the orb? Don't think we're planning to use tuist in any other SDK?

Reply:
My reasoning was that at some point (might not be that close) we will drop PHC cocoa pods support. There's no need for us to use it internally if the hybrids support dynamic frameworks.

I need to double-check if there are some limitations, but ideally there won't be 😓